cmd/import/import.go

```diff
@@ -0,0 +1,223 @@
+/*
+Tool to import pairs of files from an input sqlite database to another database.
+
+Usage: import <path-to-origin.db> <path-to-destination.db>
+
+The origin database is assumed to have the following table:
+CREATE TABLE files (name_a TEXT, name_b TEXT, content_a TEXT, content_b TEXT);
+*/
+package main
+
+import (
+	"crypto/md5"
+	"database/sql"
+	"fmt"
+	"log"
+	"os"
+
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/pmezard/go-difflib/difflib"
+)
+
+const defaultExperimentID = 1
+
+func main() {
+	args := os.Args
+
+	if len(args) != 3 {
+		printHelp()
+		return
+	}
+
+	originPath := args[1]
+	destPath := args[2]
+
+	if _, err := os.Stat(originPath); os.IsNotExist(err) {
+		log.Fatalf("File %q does not exist", originPath)
+	}
+
+	originDB, err := sql.Open("sqlite3", originPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer originDB.Close()
+
+	destDB, err := sql.Open("sqlite3", destPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer destDB.Close()
+
+	if err = bootstrap(destDB); err != nil {
+		log.Fatal(err)
+	}
+
+	if err = initialize(destDB); err != nil {
+		log.Fatal(err)
+	}
+
+	success, failures, err := importFiles(originDB, destDB)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Printf("Imported %v file pairs successfully\n", success)
+
+	if failures > 0 {
+		fmt.Printf("Failed to import %v file pairs\n", failures)
+	}
+}
+
+func printHelp() {
+	fmt.Println(`Usage: import <path-to-origin.db> <path-to-destination.db>
+
+Both arguments refer to SQLite databases. If the destination file does not
+exist, it will be created.
+The destination database does not need to be empty, new imported file pairs can
+be added to previous imports.`)
+}
+
+// bootstrap creates the necessary tables for the output DB. It is safe to call on a
+// DB that is already bootstrapped.
+func bootstrap(db *sql.DB) error {
+	_, err := db.Exec(`
+		CREATE TABLE IF NOT EXISTS users (
+			id INTEGER, github_username TEXT, auth TEXT, role INTEGER,
+			PRIMARY KEY (id))`)
+
+	if err != nil {
+		return err
+	}
+
+	_, err = db.Exec(`
+		CREATE TABLE IF NOT EXISTS experiments (
+			id INTEGER, name TEXT UNIQUE, description TEXT,
+			PRIMARY KEY (id))`)
+
+	if err != nil {
+		return err
+	}
+
+	// TODO: consider a unique constrain to avoid importing identical pairs
```

Comment:
Why not adding it?
```UNIQUE(hash_a, hash_b)```

Reply:
I would wait until we have real input examples. Maybe two pairs with the names and the same hashes point to identical files in different repos.